en/core-libraries/email.rst

```diff
@@ -42,9 +42,6 @@ add more recipients to their respective field::
     $email->to('test@example.com', 'ToTest Example');
     // The email's To recipient is: test@example.com
 
-.. deprecated:: 3.4.0
-    Use ``setFrom()``, ``setTo()``, ``setCc()`` , ``setBcc()``  and ``setSubject()`` instead.
```

Comment:
Update code samples with new method names in whole `email.rst` before removing deprecated tags otherwise it might be hard locate all method calls which need to be updated.